catalogue/webapp/utils/works.ts

```diff
@@ -246,12 +226,14 @@ export const getArchiveLabels = (work: Work): ArchiveLabels | undefined => {
   return undefined;
 };
 
-export const getCardLabels = (work: Work): Label[] => {
-  const cardLabels = [{ text: work.workType.label }];
-  if (isAvailableOnline(work)) {
-    return [...cardLabels, { text: 'Online', labelColor: 'white' }];
-  } else {
-    return cardLabels;
+export const getCardLabels = (work: Work): Label[] | undefined => {
+  if (work.workType) {
+    const cardLabels = [{ text: work.workType.label }];
+    if (isAvailableOnline(work)) {
+      return [...cardLabels, { text: 'Online', labelColor: 'white' }];
+    } else {
+      return cardLabels;
+    }
```

Comment:
This feels like it solves our immediate problem, but now a work can only be available online if it has a work type – is that coupling intentional?

I would lean towards this returning a list which can contain zero or more labels, and conditioning on whether it's non-empty rather than defined.

Reply:
Indeed 👍  https://github.com/wellcomecollection/wellcomecollection.org/pull/9468/commits/ee2e390756c5993496a35085e6793c2d6d00cf48